vehicle/as/CMakeLists.txt

```diff
@@ -28,39 +31,38 @@ catkin_package(
   autoware_vehicle_msgs
 )
 
-include_directories(
-  include
-  ${catkin_INCLUDE_DIRS}
-)
-
 add_executable(ssc_interface
   src/ssc_interface/ssc_interface_node.cpp
   src/ssc_interface/ssc_interface.cpp
 )
-target_link_libraries(ssc_interface ${catkin_LIBRARIES})
-add_dependencies(ssc_interface ${catkin_EXPORTED_TARGETS})
+ament_target_dependencies(ssc_interface ${AS_DEPENDENCIES})
+ament_export_dependencies(${AS_DEPENDENCIES})
+
 
 add_executable(pacmod_interface
   src/pacmod_interface/pacmod_interface.cpp
   src/pacmod_interface/pacmod_interface_node.cpp
 )
-target_link_libraries(pacmod_interface ${catkin_LIBRARIES})
-add_dependencies(pacmod_interface ${catkin_EXPORTED_TARGETS})
+ament_target_dependencies(pacmod_interface ${AS_DEPENDENCIES})
+ament_export_dependencies(${AS_DEPENDENCIES})
```

Comment:
Since this is not specific to a target, this looks redundant with the first call, but it's probably fine to leave it in.

Reply:
replaced by ament_auto.